docs/release/Betaflight-4.5-Release-Notes.md

```diff
@@ -0,0 +1,284 @@
+
+Angle Mode is now 'earth referenced' by default.  This means that a pure yaw stick input, while pitched forward, will result in a perfectly coordinated turn.  The code mixes in exactly the right amount of roll so that the horizon stays 'level' in the camera.  It also helps stabilise the quad during fast yaw inputs in Angle mode.  
+
+Roll inputs in angle mode will always add extra roll, and the 'horizon' in the camera will respond accordingly, if that's what the pilot wants to achieve.  
+
+The earth referencing behaviour can be disabled with `set angle_earth_ref = 0`, or its strength halved with `set angle_earth_ref = 50`.  Whoop racers may well prefer to disable it, however most beginners, and anyone doing cinematic shooting in Angle mode, should find it really nice.
+
+Angle and Horizon mode motor control is now significantly smoother, due to filtering refactoring and optimisation, reducing motor heat and minimising camera jello.
+
+Horizon mode has been changed a lot.  Horizon mode provides self-levelling when the sticks are centered and the quad is close to being flat, but flies like acro at higher stick angles or when the quad is steeply angled.  The 'self-levelling' strength, when the sticks are in the center region and the quad is nearly flat, can be as strong, or as gentle, as the pilot likes.  The angle of the frame at which there is no self-levelling can als be adjusted.  With the default settings, flips and rolls to be performed, and with a bit of tweaking, inverted hangs are possible.
+
+For more information, and sample configuration snippets, see [PR 12231](https://github.com/betaflight/betaflight/pull/12231)
+
+Thanks @ChrisRosser and @ctzsnooze
+
+## 8. Failsafe changes
+
+Failsafe indicators at the time on Rx loss are slightly different, and some safety related issues have been fixed.
+
+If `RXLOSS` is triggered by 100ms of no valid data, the message will appear in the OSD for a minimum period of 1.0s, instead of clearing immediately.  This is consistent with the arming block after Rx loss, which persists for 1.0s by default, even a brief loss of signal.  
+
+The change prevents a potentially dangerous arming conditions which could arise if GPS Rescue was active and the Rx signal at arm time was unreliable, or the user armed then quickly disarmed.  
+
+Additionally, on restoration of signal, the `failsafe_recovery_delay` period is now 500ms, recovering twice as fast as before after signal is restored.
+
+Finally, the `BADRX` OSD message now says `NOT_DISARMED`.  This occurs when the Rx signal has recovered, or has just been detected, wbut the arming switch has been left in the Armed position.  The new message provides a better explanation to the user that they must Disarm before attempting to re-arm after signal loss.
+
+thanks @ctzsnooze
+
+## 9. Dimmable RPM Harmonics
+
```

Comment:
Notch filters in Betaflight attenuate the noise right down to 0. This can be seen when comparing the gyro scaled to filtered gyro where at stock settings the motor noise band and harmonics are usually completely gone.

Sometimes we may find that the noise from RPM harmonics 2 or 3 are very low, and thus we have an opportunity to reduce filter latency and phase shift (the negative impacts of filtering) by applying less filtering to these areas. 

For example in many tri-blade situations, the second RPM harmonic can be relatively low in amplitude.  

With this feature we can now set the RPM filter 'weights' of any of the RPM Harmonics. Weight being the amount of the RPM filtering applied from a maximum of 100 (fully on) to 0 (fully off). This also means we can run the first and third harmonics only which wasn't possible previously. 

For example, `set rpm_filter_weights = 100, 0, 80` effectively disables the second harmonic and only attenuates the third harmonic by 20%.  The number sequence here: rpm_filter_weights = 100, 0, 80 denotes RPM filter 1,2,3 weights. Default is set rpm_filter_weights = 100, 100, 100.

This is an advanced tuning option because the motor noise harmonics may not be easily visible in Blackbox Log Explorer, but having the RPM filters fully active can still be having a useful effect.  Small amounts of residual motor noise in the post filter gyro can cause a large effect downstream in the Dterm. For this reason we recommend judging the impact of the rpm filter harmonics carefully using the frequency PSD spectrogram in PID toolbox. This has the best sensitivity to get an accurate idea of the tuning of this feature. 

If in doubt, when wanting to fine-tune RPM filters, we recommend sticking with 3 RPM Harmonics and starting with reducing the Q from 500 default to a max of Q1000. Following this you can consider to experiment with this feature, by reducing the weights of the 2nd and 3rd harmonic individually, and assess the impact on overall noise.

Reply:
thanks, that's nicer.  I integrated some of your suggestions verbatim, and used other parts to improve the text more generally.